js/interface/add-markers.js

```diff
@@ -320,16 +320,29 @@ Fliplet.InteractiveMap.component('add-markers', {
       this.allMarkerStyles.push(newItem)
       this.saveData()
     },
-    deleteMarkerStyle(index) {
+    deleteMarkerStyle(index, marker) {
```

Comment:
Use `options` as the parameter. `options` will then be an `Object`:
```js
{
  index: 2,
  marker: { ... }
}
```
Update function's code accordingly.

Reply:
fixed